dvc/remote/local.py

```diff
@@ -144,8 +143,9 @@ def isdir(path_info):
     def getsize(path_info):
         return os.path.getsize(fspath_py35(path_info))
 
-    def walk(self, path_info):
-        return dvc_walk(path_info, self.repo.dvcignore)
+    def walk_files(self, path_info):
+        for fname in walk_files(str(path_info), self.repo.dvcignore):
```

Comment:
`str` will return relpath (see dvc/path_info.py), which is incorrect. `walk_files` is already able to handle path_info (see dvc/utils/__init__.py), so
```suggestion
        for fname in walk_files(path_info, self.repo.dvcignore):
```
should work as is